### DISCUSSION TITLE
Sorting or Grouping of Workflows in the Actions area

### DISCUSSION THREAD

**Discussion by freddydk** [community/community#62553] [Actions]
### Select Topic Area

Product Feedback

### Body

When navigating to the Actions tab in my repository on GitHub.com, workflows are listed in alphabetic order and there is a link called **All workflows** on top.

If you have more than 10 workflows, the remaining workflows disappear behind a **Show more workflows...** link.
But... I don't think there is any way to sort or group the workflows in the list - in order to have the least used workflows in a group or like.

It would be nice to have a grouping (maybe by simply having folders under .github/workflows) or having a way of grouping workflows in the name property (name: Category/My Workflow)

**Comment by swginfinity** [comment]
yep, this has become a problem for us so we're looking at other ci/cd tools

**Reply by salemhadadi1113** [reply]
Google play

**Comment by salemhadadi1113** [comment]
Salem

**Comment by JosipPomer** [comment]
Whats up with this? 

Seems there isn't much interest, however this is really good QoL feature. Also looks like it's kinda simple to implement.
